src/D2L.Bmx/Okta/OktaClient.cs

```diff
@@ -137,13 +144,13 @@ await resp.Content.ReadAsStreamAsync(),
 		if( authnResponse?.SessionToken is not null ) {
 			return new AuthenticateResponse.Success( authnResponse.SessionToken );
 		}
-		throw new BmxException( "Error verifying MFA with Okta." );
+		return new AuthenticateResponse.Failure( resp.StatusCode );
```

Comment:
Similar in spirit to the comment above.
This one isn't as bad though, because it didn't make the code more complex. But changing anyway, at least for consistency.